docs/the-button.md

```diff
@@ -131,3 +132,20 @@ A script that gets triggered every time The Button is pushed down. By default it
 The counter part of `down.sh` - it gets triggered every time The Button is released and by default it is stopping the LED flashing initiated in the down event.
 ### [`common.sh`](https://github.com/BlokasLabs/pisound/blob/master/scripts/common/common.sh)
 Defines APIs shared between scripts for Pisound, such as flash_leds and periodic_led_blink which are used by other scripts for feedback.
+
+## Command Line Options
+
+Current command line options are always available using the `--help` argument.  The options are:
+| Option | Description |
```

Comment:
An empty line before `| Option | Description |` is necessary, otherwise with no space between the lines, it's considered as part of the same paragraph.